src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+        Random randomNum = new Random();
+
+        randomBall.number = randomNum.nextInt(101);
+        randomBall.color = randomColor.getRandomColor();
+
+        return randomBall;
+    }
+
+
+}

```

Comment:
It's better to avoid multiple instances of `Random` and `ColorSupplier` if you will need to call `getRandomBall()` method multiple times. Consider making `randomNum` and `randomColor` instance variables of the `Lottery` class to reuse them.